### ISSUE TITLE
The 1.0.0.4 release is actually 1.0.0.3

### ISSUE THREAD

**Issue by Hay1tsme** [naseweis520/osu-Sync#20]
When I downloaded the zip file from the release and replaced the old files, it still was 1.0.0.3

**Comment by muekoeff** [comment]
Really? That's pretty weird. Guess I made a mistake when zipping the files.
However, I reset the updater version to 1.0.0.3 to prevent the application to display notifications.
Actually, I already tried to migrate the whole update system to GitHub which would make it 
a) more comfortable for me
b) faster and way securer for the user.
But because of the high TLS-security-level I would have to set the application framework version higher. What a bummer.

So, as always, thank you very much for reporting this and all those other bugs :D. You're really helping me out.
